packages/contentstack-migration/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@contentstack/cli-migration",
-  "version": "1.6.1",
+  "version": "1.6.2",
   "author": "@contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
     "@contentstack/cli-command": "~1.3.0",
-    "@contentstack/cli-utilities": "~1.7.1",
+    "@contentstack/cli-utilities": "~1.7.2",
```

Comment:
@sunil-lakshman , for internal dependency we don't need to do version bump. pls check with @cs-raj